examples/simple_key_value_store.rs

```diff
@@ -184,8 +184,10 @@ impl Sendable for TestData {
 
     fn serialised_contents(&self) -> Vec<u8> {
         let mut e = cbor::Encoder::from_memory();
-        e.encode(&[&self]).unwrap();
-        e.into_bytes()
+        match e.encode(&[&self]) {
+            Ok(()) => e.into_bytes(),
+            Err(_) => Vec::<u8>::new()
+        }
```

Comment:
As written your correct. I think it's another task though to update the example to return a Result. I was only tasked to remove unwrap's, maybe this one could have been left as it was, it probably is overkill, and since it should never really fail an empty vector will rarely get returned.